pkg/privatekey/kms.go

```diff
@@ -61,3 +61,39 @@ func (s *KeyServer) CreateKeyRingIfNotExists(ctx context.Context, projectID, loc
 	// Key ring already exists.
 	return fetchedKeyRing, nil
 }
+
+// CreateKeyIfNotExists create a key if it doesn't exist.
```

Comment:
The name and the comment do not explain what happens if the key does exist. Maybe name it GetOrCreateKey()?

Reply:
updated.